main.go

```diff
@@ -26,6 +26,7 @@ type config struct {
 	DepWarnning  string
 	Script       []string `yaml:"script"`
 	Build        []string `yaml:"build"`
+	Runners      []string `yaml:"runners"`
```

Comment:
I would just rather make a change now than have to deprecate it in the near future.